ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -3265,6 +3266,46 @@ def test094_select_large_number_of_test_units_in_one_testplan(self):
                 for testunit in testunit_names:
                     self.assertIn(testunit, result['test_units'])
 
+                  
+    @parameterized.expand(['Name','No','Name:No'])
+    def test095_change_contact_config(self,search_by):
+        '''
+         Orders: Contact configuration approach: In case the user
+         configures the contact field to display name & number this action
+         should reflect on the order active table
+         LIMS-6632
+        :param search_by:
+        :return:
+        '''
+        self.contacts_page = Contacts()
+        self.info('get random contact')
+        contacts_response,_ = ContactsAPI().get_all_contacts(limit=10)
+        self.assertEqual(contacts_response['status'], 1)
+        payload = random.choice(contacts_response['contacts'])
+        self.orders_page.open_order_config()
+        self.info('change contact view by options')
+        self.contacts_page.open_contact_configurations_options()
+        if  search_by == 'Name':
+            search_text = payload['name']
+            result = payload['name']
+            search_by = [search_by]
+        elif search_by== 'No':
+            search_text = 'No: '+ str(payload['companyNo'])
+            result = str(payload['companyNo'])
+            search_by = [search_by]
+        elif search_by == 'Name:No':
+            search_text = payload['name'] + ' No: ' + str(payload['companyNo'])
+            search_by = search_by.split(':')
+            result = payload['name']
+
+        self.contacts_page.select_option_to_view_search_with(view_search_options=search_by)
+        self.info('go to order active table')
+        self.orders_page.get_orders_page()
+        self.orders_page.filter_by_contact(filter_text=result)
+        rows = self.orders_page.result_table()
```

Comment:
here you need to make sure that value assigned to cell contact exactly equal (name, no, name+no) 
so you need to use :
https://github.com/modeso-open-source/1lims-automation/blob/4b510411856316f9b3e5ae4715a4d2a9a1d7558a/ui_testing/pages/base_selenium.py#L671